horton/scripts/wpart.py

```diff
@@ -77,25 +76,26 @@ def wpart_slow_analysis(wpart, mol):
         # Prepare solid harmonics on grids.
         grid = wpart.get_grid(index)
         if wpart.lmax > 0:
-            work = np.zeros((grid.size, npure-1), float)
-            work[:,0] = grid.points[:,2] - wpart.coordinates[index,2]
-            work[:,1] = grid.points[:,0] - wpart.coordinates[index,0]
-            work[:,2] = grid.points[:,1] - wpart.coordinates[index,1]
+            work = np.zeros((grid.size, npure - 1), float)
```

Comment:
I would push for it though... Pycharm does it automatically when it's reformatting the file so if you want the rest of the auto formatting, you need to take it too. There might be a way to disable that particular auto-format, but it would mean every new developer using pycharm would have to configure it going forward...

Also it does look a bit cleaner?